### ISSUE TITLE
Casing issue in uri.path between formatted diagnostics and hoverProvider

### ISSUE THREAD

**Issue by kevinramharak** [yoavbls/pretty-ts-errors#100]
**Describe the bug**
I noticed that I did not get any prettified errors when working from a `.vue` file.

I setup a basic snippet like:
```vue
<script setup lang="ts">
import { RouterView } from 'vue-router';

const x = [1, 2, 3, '4'];

function y(param: number[]) {

}

y(x); 

</script>
<template>
    <RouterView />
</template>
````

And per screenshot the highlighter result:
![image](https://github.com/yoavbls/pretty-ts-errors/assets/14903712/e171204a-56c6-4412-bd43-eefa06fb0a54)

**Expected behavior**
I expected to see a prettified error message as vue support is implemented.

**Cause**
I cloned the repo and ran it with the debugger and found that this issue is caused by the following:

https://github.com/yoavbls/pretty-ts-errors/blob/b388ac534847a809d78c49211b023b2fb39c33f8/src/extension.ts#L72

On this line the value of `uri.path` is: `'/c:/Users/kevin/Documents/Projects/phavuer-playground/src/App.vue'`

When in the `hoverProvider.ts`:
https://github.com/yoavbls/pretty-ts-errors/blob/b388ac534847a809d78c49211b023b2fb39c33f8/src/provider/hoverProvider.ts#L6-L10

The value of `document.uri.path` is `'/C:/Users/kevin/Documents/Projects/phavuer-playground/src/App.vue'`.

Notice the casing difference of the `C` drive:
```diff
- '/c:/Users/kevin/Documents/Projects/phavuer-playground/src/App.vue'
+ '/C:/Users/kevin/Documents/Projects/phavuer-playground/src/App.vue'
```

This causes the `hoverProvider` to think there are no stored items to show, and thus show no results.

**Fix**
VS Code seems to use [this](https://github.com/microsoft/vscode-uri) implementation for the `Uri` class.

Maybe use `fsPath` instead as [it states](https://github.com/microsoft/vscode-uri/blob/6dec22d7dcc6c63c30343d3a8d56050d0078cb6a/src/uri.ts#L180-L204):
```js
/**
* Returns a string representing the corresponding file system path of this URI.
* Will handle UNC paths, normalizes windows drive letters to lower-case, and uses the
* platform specific path separator.
*
* * Will *not* validate the path for invalid characters and semantics.
* * Will *not* look at the scheme of this URI.
* * The result shall *not* be used for display purposes but for accessing a file on disk.
*
*
* The *difference* to `URI#path` is the use of the platform specific separator and the handling
* of UNC paths. See the below sample of a file-uri with an authority (UNC path).
*/
```

**Comment by yoavbls** [comment]
Thank you for the detailed issue!
I think the main reason it didn't work is that In the new official extension of [Vue VSCode plugin](https://marketplace.visualstudio.com/items?itemName=Vue.volar), Hybrid mode is enabled by default, which means that a TypeScript LSP Plugin publishes the diagnostics as `ts-plugin` instead of `ts`.
I fixed it in #103, published in v0.5.4

**Comment by sentience** [comment]
@yoavbls Looks like v0.5.4 still needs to be merged into main (or perhaps you just haven't pushed your changes to main)?
